.eslintrc.js

```diff
@@ -19,7 +19,7 @@ const majorMinorRegExp = escapeRegExp( version.replace( /\.\d+$/, '' ) ) + '(\\.
 module.exports = {
 	root: true,
 	extends: [
-		'./eslint/config.js',
+		'@wordpress/eslint-config-default',
```

Comment:
This should be `@wordpress/eslint-config-gutenberg`

Reply:
Why those configs are different? let's talk about them on meetings and align them instead of adding a "gutenberg" specific package. since we merged all the packages here, this repository is not Gutenberg specific anymore

Reply:
Yes, let's talk on Tuesday about this. I'm also not happy about including Gutenberg in the name. We should also discuss what is the future plan for maintaining all the existing configurations.

Reply:
I added it to the agenda: https://docs.google.com/document/d/1JjIk7I_PD7CAU7rhYEPNUza2thQVZI1RlukWEWGlYLc/edit#.